custom.cmake cannot install lvgl.pc

### LVGL version

v8.4

### What happened?

When I issue sudo make install on raspberry pi I get the error 
`CMake Error at _deps/lvgl-build/cmake_install.cmake:70 (file):
  file INSTALL cannot find "/home/pi/sdrberry/build/lvgl.pc": No such file or
  directory.
Call Stack (most recent call first):
  cmake_install.cmake:47 (include)
`
I don't use this file and also don't know why it is necessary, can it be removed? Or issue fixed. Because cmake will stop and don't install the application.

### How to reproduce?
In CMakeList.txt I fetch lvgl with 
`FetchContent_Declare(lvgl GIT_REPOSITORY  https://github.com/lvgl/lvgl.git GIT_TAG v8.4.0)`
Install with:
`sudo make install`

I made a small project which demonstrates the issue.  
You can clone it from https://github.com/paulh002/cmaketest  

to build it:  
git clone https://github.com/paulh002/cmaketest  
cd cmaketest
mkdir build
cd build
cmake ..
make
sudo make install


Result will be:
pi@pi5:~/cmaketest/build $ sudo make install
[ 45%] Built target lvgl
[ 45%] Built target cmaketest
[ 85%] Built target lvgl_examples
[100%] Built target lvgl_demos
Install the project...
-- Install configuration: ""
-- Installing: /usr/local/include/lvgl/../lv_conf.h
CMake Error at _deps/lvgl-build/cmake_install.cmake:70 (file):
  file INSTALL cannot find "/home/pi/cmaketest/build/lvgl.pc": No such file
  or directory.
Call Stack (most recent call first):
  cmake_install.cmake:47 (include)


make: *** [Makefile:100: install] Error 1

--- a/env_support/cmake/custom.cmake
+++ b/env_support/cmake/custom.cmake
@@ -99,6 +99,8 @@ if("${INC_INSTALL_DIR}" STREQUAL "")
   set(INC_INSTALL_DIR "include/lvgl")
 endif()
 
+set(CMAKE_INSTALL_PREFIX .)
+
 #Install headers
 install(
   DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/src"
@@ -131,8 +133,8 @@ if(NOT LV_CONF_BUILD_DISABLE_DEMOS)
 endif()
 
 
-configure_file("${LVGL_ROOT_DIR}/lvgl.pc.in" lvgl.pc @ONLY)
-configure_file("${LVGL_ROOT_DIR}/lv_version.h.in" lv_version.h @ONLY)
+configure_file("${LVGL_ROOT_DIR}/lvgl.pc.in" ${CMAKE_BINARY_DIR}/lvgl.pc @ONLY)
+configure_file("${LVGL_ROOT_DIR}/lv_version.h.in" ${CMAKE_BINARY_DIR}/lv_version.h @ONLY)
 
 install(
   FILES "${CMAKE_CURRENT_BINARY_DIR}/lvgl.pc"

